testing/tools/integration/observability.py

```diff
@@ -46,13 +57,13 @@ class ObservabilityResponseError(Exception):
     'state-entity-query': 'state-entity-query'}
 
 
-def external_sender_query(host, port, query_type):
+def external_sender_query(addr, query_type):
```

Comment:
Naming: it's unclear whether this function constructs a query to be used later, or actually performs it.
Maybe something like `perform_observability_query` would work better?

Reply:
I partly agree, but it's only used internally in `observability` at the moment, and within that context it does what it says: execute a query using `external_sender`. I'll think about it some more, but for now I don't have any good ideas of how to change this in a meaningful way.

Reply:
Gotcha. Agreed.